### ISSUE TITLE
Exporting to PDF generates an `java.lang.NoSuchMethodError` error

### ISSUE THREAD

**Issue by tukanos** [freeplane/freeplane#2096]
**Describe the bug**
Exporting mind map to PDF ends up in error

**To Reproduce**
1. Go to `File`
2. Click on `Export map`
3. In `Files of Type` select `Portable Document Format (PDF) (.pdf)
4. Click on Save button

**Expected behavior**
Generate PDF without errors.

**Mind map files**
[testing_mind_map.zip](https://github.com/user-attachments/files/17617654/testing_mind_map.zip)

**Screenshots**
See the below log output.

**Desktop (please complete the following information):**
 - Freeplane version: 1.12.6 (latest stable)
 - OS: Debian
 - Version: Testing - Trixie
 - Java version: `OpenJDK Runtime Environment Zulu11.68+17-CA (build 11.0.21+9-LTS)`

**Additional context**

Error generated:

```Java
java.lang.NoSuchMethodError: 'java.util.List org.apache.fontbox.cff.CFFParser.parse(byte[])'
	at org.apache.fop.fonts.truetype.OTFFile.initializeFont(OTFFile.java:103)
	at org.apache.fop.fonts.truetype.OpenFont.readFont(OpenFont.java:822)
	at org.apache.fop.fonts.truetype.OFFontLoader.read(OFFontLoader.java:119)
	at org.apache.fop.fonts.truetype.OFFontLoader.read(OFFontLoader.java:103)
	at org.apache.fop.fonts.FontLoader.getFont(FontLoader.java:126)
	at org.apache.fop.fonts.FontLoader.loadFont(FontLoader.java:110)
	at org.apache.fop.fonts.LazyFont.load(LazyFont.java:121)
	at org.apache.fop.fonts.LazyFont.hasChar(LazyFont.java:181)
	at org.apache.fop.svg.font.FOPFontFamilyResolverImpl.getFamilyThatCanDisplay(FOPFontFamilyResolverImpl.java:70)
	at org.apache.fop.svg.font.FOPFontFamilyResolverImpl.getFamilyThatCanDisplay(FOPFontFamilyResolverImpl.java:34)
	at org.apache.batik.bridge.StrokingTextPainter.createModifiedACIForFontMatching(StrokingTextPainter.java:549)
	at org.apache.batik.bridge.StrokingTextPainter.computeTextRuns(StrokingTextPainter.java:224)
	at org.apache.fop.svg.NativeTextPainter.computeTextRuns(NativeTextPainter.java:224)
	at org.apache.batik.bridge.StrokingTextPainter.getTextRuns(StrokingTextPainter.java:191)
	at org.apache.batik.bridge.StrokingTextPainter.getBounds2D(StrokingTextPainter.java:1082)
	at org.apache.batik.bridge.TextNode.getPrimitiveBounds(TextNode.java:243)
	at org.apache.batik.gvt.AbstractGraphicsNode.getTransformedPrimitiveBounds(AbstractGraphicsNode.java:854)
	at org.apache.batik.gvt.AbstractGraphicsNode.getTransformedBounds(AbstractGraphicsNode.java:820)
	at org.apache.batik.gvt.CompositeGraphicsNode.getPrimitiveBounds(CompositeGraphicsNode.java:207)
	at org.apache.batik.gvt.CompositeGraphicsNode.getTransformedPrimitiveBounds(CompositeGraphicsNode.java:295)
	at org.apache.batik.gvt.AbstractGraphicsNode.getTransformedBounds(AbstractGraphicsNode.java:820)
	at org.apache.batik.gvt.CompositeGraphicsNode.getPrimitiveBounds(CompositeGraphicsNode.java:224)
	at org.apache.batik.gvt.CompositeGraphicsNode.getTransformedPrimitiveBounds(CompositeGraphicsNode.java:295)
	at org.apache.batik.gvt.AbstractGraphicsNode.getTransformedBounds(AbstractGraphicsNode.java:820)
	at org.apache.batik.gvt.CompositeGraphicsNode.getPrimitiveBounds(CompositeGraphicsNode.java:207)
	at org.apache.batik.gvt.CompositeGraphicsNode.getTransformedPrimitiveBounds(CompositeGraphicsNode.java:295)
	at org.apache.batik.gvt.AbstractGraphicsNode.getTransformedBounds(AbstractGraphicsNode.java:820)
	at org.apache.batik.gvt.CompositeGraphicsNode.getPrimitiveBounds(CompositeGraphicsNode.java:207)
	at org.apache.batik.gvt.AbstractGraphicsNode.getBounds(AbstractGraphicsNode.java:768)
	at org.apache.batik.gvt.AbstractGraphicsNode.paint(AbstractGraphicsNode.java:443)
	at org.apache.fop.svg.PDFTranscoder.transcode(PDFTranscoder.java:184)
	at org.apache.batik.transcoder.XMLAbstractTranscoder.transcode(XMLAbstractTranscoder.java:142)
	at org.apache.batik.transcoder.SVGAbstractTranscoder.transcode(SVGAbstractTranscoder.java:158)
	at org.freeplane.plugin.svg.ExportPdf.transcodeSvgToPdfFile(ExportPdf.java:96)
	at org.freeplane.plugin.svg.ExportPdf.lambda$export$0(ExportPdf.java:74)
	at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by dpolivaev** [comment]
I fixed it last week, please try preview 1.2.7_05
https://sourceforge.net/projects/freeplane/files/freeplane%20preview/

**Comment by tukanos** [comment]
> I fixed it last week, please try preview 1.2.7_05 https://sourceforge.net/projects/freeplane/files/freeplane%20preview/

I do confirm that it works! Thank you.
